packages/api/src/schema/db-schema.yaml

```
@@ -0,0 +1,1140 @@
+                    minItems: 1
+                    default:
+                      - "*"
+                    items:
+                      type: string
+                      enum:
+                        - post
+                        - get
+                        - put
+                        - patch
+                        - delete
+                        - head
+                        - options
+                        - "*"
+        lastSeen:
+          type: number
+          example: 1587667174725
+        createdAt:
+          type: number
+          readOnly: true
+          description:
+            Timestamp (in milliseconds) at which token object was created
+          example: 1587667174725
+    transcode-asset-payload:
+      additionalProperties: false
+      required:
+        - name
+        - profile
+      properties:
+        assetId:
+          type: string
+          description: ID of the asset to transcode
+        objectStoreId:
+          type: string
+          description: Object store ID where the asset is stored
+          writeOnly: true
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        name:
+          type: string
+          description: >-
+            Name of the asset. This is not necessarily the filename, can be a
+            custom name or title
+          example: filename.mp4
+        profile:
+          $ref: "#/components/schemas/ffmpeg-profile"
+    signing-key-patch-payload:
+      type: object
+      additionalProperties: false
+      properties:
+        name:
+          $ref: "#/components/schemas/signing-key/properties/name"
+        disabled:
+          $ref: "#/components/schemas/signing-key/properties/disabled"
+    suspend-user-payload:
+      type: object
+      additionalProperties: false
+      required:
+        - suspended
+      properties:
+        suspended:
+          $ref: "#/components/schemas/user/properties/suspended"
+        emailTemplate:
+          type: string
+          description:
+            Name of template to send to the users regarding the suspension.
+          enum:
+            - copyright
+    webhook:
+      table: webhook
+      properties:
+        id:
+          readOnly: true
+        kind:
+          type: string
+          example: webhook
+          readOnly: true
+        userId:
+          readOnly: true
+          type: string
+          index: true
+        createdAt:
+          readOnly: true
+        events:
+          index: true
+          indexType: gin
+        status:
+          readOnly: true
+          properties:
+            lastFailure:
+              readOnly: true
+              properties:
+                timestamp:
+                  readOnly: true
+                error:
+                  readOnly: true
+                response:
+                  readOnly: true
+                statusCode:
+                  readOnly: true
+            lastTriggeredAt:
+              readyOnly: true
```

typo (should go on api reference anyway)
```suggestion
              readOnly: true
```